src/main/java/com/google/sps/data/NSWGovAdaptor.java

```diff
@@ -0,0 +1,163 @@
+package com.google.sps.data;
+
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.time.*; 
+import java.time.format.DateTimeFormatter;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+/** File to run the GET request in java before putting it in the main adaptor function. 
+ */
+public class NSWGovAdaptor {
+
+  private HashMap<Integer, GovSiteDetails> dataMap;
+
+  public NSWGovAdaptor() throws Exception {
+    this.dataMap = new HashMap<>();
+    this.updateSiteInfo();
+  }
+
+  //To be deleted in production code, used for TestAdaptor.java
+  public HashMap<Integer, GovSiteDetails> getMap() {
+    return this.dataMap;
+  }
+
+  private Coordinates getCoord(Integer siteId) {
+
+    Coordinates siteCoord = new Coordinates(); 
+    try {
+
+      //Check if the siteId is in the map or not. If not, update the map first
+      if (this.dataMap.containsKey(siteId) == false ) {
+        this.updateSiteInfo();
+      }
+      
+      siteCoord.setCoordinates(this.dataMap.get(siteId).lng, this.dataMap.get(siteId).lat);
+
+    } catch (final NullPointerException e) {
+
+      //Indent continuation at least +4 from original line
```

Comment:
This comment no longer true? Not sure what this comment means.

Reply:
This comment was referring to the long line of code which i split into three lines - and i made it 4 lines indented from the original line of code to follow style guide and thought i'll leave it there incase you guys asked about it